base/setup/usetup/lang/ru-RU.h

```diff
@@ -161,7 +161,7 @@ static MUI_ENTRY ruRUWelcomePageEntries[] =
     {
         6,
         24,
-        "http://www.reactos.ru",
+        "https://reactos.ru",
```

Comment:
```suggestion
        "https://reactos.org",
```

[http~~s~~://reactos.ru](https://reactos.ru/) redirects to https://reactos.org but with ssl issues.

Reply:
Also exists https://reactos.org/ru redirect

Reply:
fixed as proposed by 2upvoted GGLink proposal.